### ISSUE TITLE
Monitoring of the ETL process by Logging

### ISSUE THREAD

**Issue by dominic0df** [amosproj/amos2024ss08-cloud-native-llm#10]
## User story
1. As a Data Engineer
2. I want to implement a logging system
3. So that I am able to identify problems in the dataset creation

## Acceptance criteria
* [ ] The amount of data loaded is logged (f.e. counter)
* [ ] An indicator how far the ETL process has advanced is implemented
* [ ] The cause of requests that failed and could not be retried is logged
* [ ] The cause of requests that failed and where the retry exceeded is logged

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by dominic0df** [comment]
actual size 2
